### DISCUSSION TITLE
Integration of gatsby-theme-blog and i18n

### DISCUSSION THREAD

**Discussion by ColadaFF** [gatsbyjs/themes#121] [Q&A]
Hi there,

I've been trying to integrate i18n with the new gatsby-theme-blog, but haven't been able to do so.

I've tried a lot of gatsby i18n packages.

Is there and official way to do this?

I also tried with gatsby-theme-i18n

But get the error "Cannot query field "themeI18N" on type "Query"."

Thanks!

**Comment by logemann** [comment]
most likely not needed anymore but for everyone else:

This error happens when you forget to include the gatsby-theme-i18n plugin in your config (and perhaps only added gatsby-theme-i18n-react-i18next plugin).

So just add that:

```
{
      resolve: `gatsby-theme-i18n`,
      options: {
        defaultLang: `en`,
        configPath: require.resolve(`./i18n/config.json`),
      },
    },
```

to your config section.
